integration_tests/integration/serviceProviderReferrals.cy.js

```diff
@@ -914,8 +914,8 @@ describe('Service provider referrals dashboard', () => {
         it('should present no errors and display scheduled appointment', () => {
           const tomorrow = moment('09:02:00', 'HH:mm:ss').add(1, 'days')
           cy.visit(`/service-provider/action-plan/${actionPlan.id}/sessions/1/edit/start`)
-          cy.get('#date-day').type(tomorrow.format('DD'))
-          cy.get('#date-month').type(tomorrow.format('MM'))
+          cy.get('#date-day').type(tomorrow.format('D'))
```

Comment:
So its gonna take only one digit of the Day. For example if its 28th, is it gonna take only 2?

Reply:
No, it will use the least amount of digits needed to display the day. For example:

Field | Token | Output
-- | -- | --
Day of Month | D | 1 2 ... 30 31
  | Do | 1st 2nd ... 30th 31st
  | DD | 01 02 ... 30 31

Source: https://momentjs.com/docs/#/displaying/format/

Reply:
Same for `M`:




Field | Token | Output
-- | -- | --
Month | M | 1 2 ... 11 12
  | Mo | 1st 2nd ... 11th 12th
  | MM | 01 02 ... 11 12
  | MMM | Jan Feb ... Nov Dec
  | MMMM | January February ... November December


<br class="Apple-interchange-newline">

Reply:
That's good then.